cypress/plugins/index.js

```diff
@@ -16,7 +16,7 @@
  * @type {Cypress.PluginConfig}
  */
 // eslint-disable-next-line no-unused-vars
-module.exports = (on, config) => {
+module.exports = (_on, config) => {
   config = require('cypress-dotenv')(config);
```

Comment:
you can move 
```
        cy.on('fail', () => {
             cy.screenshot();
         });
```
to here, so it happens for all your test files, not just a11y spec.

Reply:
I tried this but it looks like `fail` isn't an option that can be used in that file.

<img width="1193" alt="Screenshot 2022-02-21 at 14 52 49" src="https://user-images.githubusercontent.com/1377253/154979847-c733d20a-e42d-41b7-9876-26ab5cdbbc41.png">

Unless I'm doing something wrong.

Reply:
Yeah the cy.on('fail', ()... does work, although I do prefer to click though/inspect the issue to see what actually went wrong so I don't use it that much 🤷